src/ml_tooling/baseclass.py

```diff
@@ -471,15 +382,15 @@ def train_estimator(self) -> "ModelData":
 
         """
         logger.info("Training estimator...")
-        self.estimator.fit(self.data.x, self.data.y)
+        self.estimator.fit(data.x, data.y)
         # Prevent confusion, as train_estimator does not return a result
         self.result = None
         logger.info("Estimator trained!")
 
         return self
 
     def score_estimator(
-        self, metric: Optional[str] = None, cv: Optional[int] = False
+        self, data: DataSet, metric: Optional[str] = None, cv: Optional[int] = False
```

Comment:
We should also note that score estimator uses only train for cv and train+test if non-cv